include/CMakeLists.txt

```diff
@@ -51,7 +51,15 @@ target_sources(cpr PRIVATE
 )
 
 # Filesystem
-if ((CMAKE_CXX_COMPILER_ID STREQUAL "GNU" AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS 9.1) OR CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
+if(CPR_USE_BOOST_FILESYSTEM)
+  find_package(Boost 1.44 REQUIRED COMPONENTS filesystem)
+  if(Boost_FOUND)
+    include_directories(${Boost_INCLUDE_DIRS})
+    target_link_libraries(cpr PUBLIC ${Boost_LIBRARIES})
```

Comment:
One minor thing I just found. Please use targets. And replace the `include_directories` with `target_include_directories`

Reply:
Oh, yes. Changed it to `target_include_directories`. Thanks!